phpBB/phpbb/request/deactivated_super_global.php

```diff
@@ -103,7 +105,7 @@ public function offsetUnset($offset)
 	* @return void
 	* @psalm-suppress InvalidReturnType
 	*/
-	public function count()
+	public function count(): int
```

Comment:
Why use `int` and not `void` together with `#[ReturnTypeWillChange]` like for `getIterator()`?